mapboxgl/templates/chloropleth.html

```diff
@@ -0,0 +1,83 @@
+{% extends "main.html" %}
+
+{% block javascript %}
+var legend = document.getElementById('legend');
+
+mapboxgl.accessToken = '{{ accessToken }}';
+
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: '{{ styleUrl }}', 
+    center: {{ center }},
+    zoom: {{ zoom }},
+    transformRequest: (url, resourceType)=> {
+        return {
+           url: [url.slice(0, url.indexOf("?")+1), "pluginName=PythonMapboxgl&", url.slice(url.indexOf("?")+1)].join('')
+         }
+    }
+});
+
+map.addControl(new mapboxgl.NavigationControl());
+
+calcCircleColorLegend({{ colorStops }}, "{{ colorProperty }}");
+
+map.on('style.load', function() {
+    
+    // Add data source
+    map.addSource("data", {
+        "type": "geojson",
+        "data": {{ geojson_data }},
+        "buffer": 1,
+        "maxzoom": 14
+    });
+    
+    // Add data layer
+    map.addLayer({
+        "id": "polygons",
```

Comment:
Let's make the layer name more descriptive- `choropleth-fill`.  Then a label layer could be `choropleth-label`

Reply:
Addressed with d5a5435.

Reply:
Added "chloropleth-label" layer in 3a5b3e6.